File: fixtures/scale/csharp/large/Families/Polybius/PolybiusVectors.cs
using System.Collections.Generic;
using CipherLab.Abstractions;

namespace CipherLab.Families.Polybius
{
    public static class PolybiusVectors
    {
        public static List<TestVector> All()
        {
            return new List<TestVector>
            {
                new TestVector("ATTACKATDAWN", "DZCMRCVREEDX"),
                new TestVector("THEQUICKBROWNFOX", "WNNCJAXICVVGAVHT"),
                new TestVector("DEFENDTHEEASTWALL", "GKOQCVOFFIHCGMTHK"),
            };
        }
    }
}

File: fixtures/scale/php/large/src/Families/Polybius/PolybiusVectors.php
<?php

namespace CipherLab\Families\Polybius;

use CipherLab\Abstractions\TestVector;

// Known-answer test data captured from PolybiusCipher's own encode().
class PolybiusVectors
{
    /** @return TestVector[] */
    public static function all(): array
    {
        return [
            new TestVector("ATTACKATDAWN", "DZCMRCVREEDX"),
            new TestVector("THEQUICKBROWNFOX", "WNNCJAXICVVGAVHT"),
            new TestVector("DEFENDTHEEASTWALL", "GKOQCVOFFIHCGMTHK"),
        ];
    }
}
